packages/cloudflare-optimizer-scripts/src/index.js

```diff
@@ -259,6 +278,44 @@ function resetOptimizerForTesting() {
   ampOptimizer = null;
 }
 
+/**
+ * @description Determine whether {Response} is Redirect style
+ * @param {Response} response
+ * @returns {boolean}
+ */
+function isRedirect(response) {
+  return response.status >= 300 && response.status <= 399;
+}
+
+/**
+ * @param {!Response} response
+ * @param {!ConfigDef} config
+ * @description Rewrites the location header if the request is coming from the proxied origin, designed to work with
+ */
+function rewriteRedirectHeader(response, config) {
+  const locationHeader = response.headers.get('location');
```

Comment:
I am changing the cases of all the strings passed to `Header` to lowercase, since even though this won't matter in production. Since, we are using Map as polyfills the case-sensitivity matters there. 

I will replace this with a better poly fill later on.

Reply:
Sounds good!